timezone/test/timezone_auto_test.rb

```diff
@@ -30,13 +31,13 @@
       allow(Yast::Wizard).to receive(:CreateDialog)
       allow(Yast::Wizard).to receive(:CloseDialog)
       allow(Yast::Wizard).to receive(:HideAbortButton)
-      allow(subject).to receive(:TimezoneDialog).with(
+      allow_any_instance_of(Yast::TimezoneDialogsInclude).to receive(:TimezoneDialog).with(
         {"enable_back"=>true, "enable_next"=>true})
         .and_return(true)
     end
 
     it "runs timezone dialog" do
-      expect(subject).to receive(:TimezoneDialog).with(
+      expect_any_instance_of(Yast::TimezoneDialogsInclude).to receive(:TimezoneDialog).with(
```

Comment:
this can be fixes this tricky way or another possibility is to move Yast.include to `initialize` from `main`. That also works fine and I usually prefer this way.